lib/buffer.js

```diff
@@ -32,8 +29,10 @@ Buffer.prototype.swap16 = function swap16() {
   // do the swap in javascript. For larger buffers,
   // dropping down to the native code is faster.
   const len = this.length;
-  if (len % 2 !== 0)
-    throw new RangeError('Buffer size must be a multiple of 16-bits');
+  if (len % 2 !== 0) {
+    const errors = require('internal/errors');
+    throw new errors.RangeError('BUFFERSWAP16');
```

Comment:
Yes, it's used _25 times_ in this file.

Btw, it being required at the top of `buffer.js` probably nullifies the possible gains from lazy-loading in other places.